ui/src/lib/components/Form.svelte

```diff
@@ -287,191 +293,371 @@
+                    <label>
+                        <input type=checkbox
+                               bind:group={formConfigMap['format']._selectedFormats} value={format}
+                               on:click|stopPropagation>
+                        {format}
+                    </label>
+                {/each}
+            </div>
+        {/if}
+    </div>
+
+    <!-- STYLE SECTION -->
+    <div class="inputContainer">
+        <p class="inputContainerTitle" on:click={handleClick('style')}>
+            <input
+                    type="checkbox"
+                    bind:checked={formConfigMap['style']._active}
+                    on:click|stopPropagation
+            />
+            Style <i class="arrow {moduleUis['style'].arrowDirection}"/>
+        </p>
+        {#if moduleUis['style'].visible && isStyleConfig(formConfigMap['style'])}
+            <div class="moduleForm" transition:slide>
+                {#each formConfigMap['style'].allTypes as style}
+                    <label>
+                        <input type=checkbox
+                               bind:group={formConfigMap['style']._selectedTypes} value={style}
+                               on:click|stopPropagation>
+                        {style}
+                    </label>
+                {/each}
+            </div>
+        {/if}
+    </div>
+
+    <!-- BODY SECTION -->
+    <div class="inputContainer">
+        <p class="inputContainerTitle" on:click={handleClick('body')}>
+            <input
+                    type="checkbox"
+                    bind:checked={formConfigMap['body']._active}
+                    on:click|stopPropagation
+            />
+            Body parts <i class="arrow {moduleUis['body'].arrowDirection}"/>
+        </p>
+        {#if moduleUis['body'].visible && isBodyConfig(formConfigMap['body'])}
+            <div class="moduleForm one" transition:slide>
+                <label>
+                    <input type=checkbox bind:checked={moduleUis['body'].faceChecked}>
+                    Detect faces
+                </label>
+                <label>
+                    Minimal confidence
+                    <input
+                            type="number"
+                            name="faceWeight"
+                            bind:value={formConfigMap['weather']._faceWeight}
+                            min="1"
+                            max="100"
+                            step="1"
+                            disabled={!moduleUis['body'].faceChecked}/>
+                </label>
+
+                <label>
+                    <input type=checkbox bind:checked={moduleUis['body'].handsChecked}>
+                    Detect hands
+                </label>
+                <label>
+                    Minimal confidence
+                    <input
+                            type="number"
+                            name="handsWeight"
+                            bind:value={formConfigMap['weather']._handsWeight}
```

Comment:
Same here as above, 'body'. Also we could name the variable _handsWeight as _handsConfidence as it is in gui or vice versa - put Hands Weight in gui.